smartapps/smartthings/logitech-harmony-connect.src/logitech-harmony-connect.groovy

```diff
@@ -339,7 +339,7 @@ def initialize() {
 	state.aux = 0
 	if (selectedhubs || selectedactivities) {
 		addDevice()
-        runEvery5Minutes("discovery")
+        runEvery5Minutes("poll")
```

Comment:
Is there more details about what the goal of this change is?  Because switching from one method call to another seems to accomplish different goals, because it looks like discovery initially hits a completely different API compared to what gets hit by poll(), and then discovery() calls poll() on it's own once it goes through it's own initial API call.  I just want to make sure that some of that logic isn't necessary that is now being excluded by calling a different method.